features/tabelwaarden.feature

```diff
@@ -43,23 +43,23 @@ De onderstaanden tabellen met de beschreven waarden worden geleverd door de "BRP
     | F                                                | fout                                 |
     | .                                                | onbekend                             |
 
-  Voor de enumeratie SoortVerbintenis wordt de tabel met identificatie "soort_verbintenis" opgenomen.
+  Voor de enumeratie SoortVerbintenis wordt de tabel met identificatie "Soort_Verbintenis" opgenomen.
   In de omschrijving wordt de volgende tekst opgenomen : "Soort verbintenis die bij de burgerlijke stand is ingeschreven"
   In de waardetabel worden de volgende waarden opgenomen:
     | code                                             | omschrijving                         |
     | ------------------------------------------------ | ------------------------------------ |
     | H                                                | huwelijk                             |
     | P                                                | geregistreerd partnerschap           |
 
-    Voor de enumeratie SoortAdres wordt de tabel met identificatie "soort_adres" opgenomen.
+    Voor de enumeratie SoortAdres wordt de tabel met identificatie "Soort_Adres" opgenomen.
```

Comment:
waarom eigenlijk "soort" van het adres? Het gaat om de functie van het adres. De soort van het adres is opgenomen in "type" (soort en type zijn volgens mij synoniemen).